File: iOS/Swift/SampleSwift/Models/SensorStatus.swift
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license.

enum SensorStatus {
    /// The sensor's status is unknown
    case Indeterminate

    /// Access has not been granted by the user
    case Blocked

    /// The sensor is active but has not provided any observations
    case Unavailable

    /// The sensor is providing observations
    case Available
}

File: Unity/Assets/External/AzureSpatialAnchors.Examples/Scripts/SensorStatus.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license.

public enum SensorStatus
{
    /// <summary>
    /// GeoLocation data is available.
    /// </summary>
    Available,
    /// <summary>
    /// GeoLocation was disabled in the SensorCapabilities.
    /// </summary>
    DisabledCapability,
    /// <summary>
    /// No sensor fingerprint provider has been created.
    /// </summary>
    MissingSensorFingerprintProvider,
    /// <summary>
    /// No GPS data has been received.
    /// </summary>
    NoData
}